AgriParcel/schema.json

```diff
@@ -152,6 +152,38 @@
                             "$ref": "https://smart-data-models.github.io/data-models/common-schema.json#/definitions/EntityIdentifierType"
                         }
                     ]
+                },
+                 "soilTextureType": {
+                    "type": "string",
```

Comment:
We need a minimal description of the attribute (besides the enumeration, which is great)